doc/modules/mod_muc_commands.md

```diff
@@ -0,0 +1,47 @@
+# MongooseIM's multi-user chat commands set
+
+## Purpose
+This is a set of commands, which provide actions connected with multi-user chat features.
```

Comment:
This is a set of commands, providing actions connected with multi-user chat features?